packages/babel-parser/ast/spec.md

```diff
@@ -1259,6 +1261,18 @@ interface ImportNamespaceSpecifier <: ModuleSpecifier {
 
 A namespace import specifier, e.g., `* as foo` in `import * as foo from "mod.js"`.
 
+### ImportModuleAttribute
+
+```js
+interface ImportModuleAttribute <: Node {
+  type: "ImportModuleAttribute";
+  key: Identifier;
+  value: StringLiteral;
```

Comment:
Does it mean that tc39 has decided it will only support string types primitive?

Reply:
Probably not, but it's better to be conservative for now. Allowing new things is "less breaking" than disallowing other kinds of literals.

Reply:
It's still in discussion. I would ideally support all the primtives, but Babel could get keep it simple for now

Reply:
We're still discussing this among the people working on module attributes. I'd rather hold off on landing this a bit longer.